sass/blocks/wrapper/_wrapper.scss

```diff
@@ -0,0 +1,11 @@
+//Desktop screen
+.wrapper {
+  @media (min-width: $desktop-size) {
+    width: 100%;
+    max-width: 120rem;
+    margin: 0 auto;
+    position: relative;
+    box-shadow: 0 1px 80px rgba(0, 0, 0, 0.2);
+    background-color: $color-white;
+  }
```

Comment:
Ok, let's refactor a bit:
1) You actually don't need media query here at all
2) it's going to be 100% anyway as this is the default behaviour of a block div, but we can keep to make it explicitly if you want 
3) why do you need position relative here? and it should be at the top of all the properties if you need it
4) still think that blur radius is too big, the shadow is smaller in designs 
I think there is a way to convert photoshop shadow to CSS, the first link I googled try it http://www.melanieceraso.com/psd-to-css3/
5) put background-color property above box-shadow.